### PR TITLE
Models for address register from voters register

### PR DESCRIPTION
**Pull request by IegorSoboliev**
please, look at models in the new register. If this is ok, I will add needed migrations then

### CODE REVIEW

File: location_register/models/drv_models.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        return self.name
+
+
+class Street(DataOceanModel):
+    region = models.ForeignKey(Region, on_delete=models.CASCADE)
+    district = models.ForeignKey(District, on_delete=models.CASCADE)
+    council = models.ForeignKey(Council, on_delete=models.CASCADE)
+    ato = models.ForeignKey(Ato, on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+    previous_name = models.CharField(max_length=100)
```
Comment (Anntroy) [inline location_register/models/drv_models.py:54]: Is previous_name historical data?

File: location_register/models/drv_models.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+from data_ocean.models import DataOceanModel
+
+
+class Region(DataOceanModel):
+    code = models.IntegerField(max_length=3, unique=True)
+    number = models.IntegerField(max_length=3, unique=True)
+    name = models.CharField(max_length=30, unique=True)
+    short_name = models.CharField(max_length=5, unique=True)
+    capital = models.CharField(max_length=15, unique=True, null=True)
```
Comment (Anntroy) [inline location_register/models/drv_models.py:11]: Does 'capital'  mean the region's main city? Why null=True?

Comment (IegorSoboliev) [reply location_register/models/drv_models.py:11]: Kyiv and Sevastopol are regions without capitals

File: location_register/models/drv_models.py (hunk 1 above)
Comment (IegorSoboliev) [reply location_register/models/drv_models.py:54]: No

File: location_register/models/drv_models.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+class District(DataOceanModel):
+    region = models.ForeignKey(Region, on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+
+class Council(DataOceanModel):
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply location_register/models/drv_models.py:25]: an example - "Київська міська рада"

File: location_register/models/drv_models.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
+from django.db import models
+
+from data_ocean.models import DataOceanModel
+
+
+class Region(DataOceanModel):
+    code = models.IntegerField(max_length=3, unique=True)
+    number = models.IntegerField(max_length=3, unique=True)
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply location_register/models/drv_models.py:8]: both fields are invented by Central Election Commission)

**Pull request merged**